pkg/engine/engine_test.go

```diff
@@ -214,6 +222,140 @@ func TestEngine_DuplicatSecrets(t *testing.T) {
 	assert.Equal(t, want, e.GetMetrics().UnverifiedSecretsFound)
 }
 
+// TestEngine_VersionedDetectorsVerifiedSecrets is a test that detects ALL verified secrets across
+// versioned detectors.
+func TestEngine_VersionedDetectorsVerifiedSecrets(t *testing.T) {
```

Comment:
BTW this breaks `make test-community` since the secrets aren't accessible.

```
--- FAIL: TestEngine_VersionedDetectorsVerifiedSecrets (0.01s)
    engine_test.go:231:
                Error Trace:    /tmp/trufflehog/pkg/engine/engine_test.go:231
                Error:          Received unexpected error:
                                failed to create secretmanager client: google: could not find default credentials. See https://cloud.google.com/docs/authentication/external/set-up-adc for more information
                                github.com/trufflesecurity/trufflehog/v3/pkg/common.GetSecret
                                        /tmp/trufflehog/pkg/common/secrets.go:55
```

Reply:
Likely - yes. The beginning of the trace looks the same. The aftermath is a panic and then segfault.